Exiled.Events/Patches/Events/Player/SpawningRagdoll.cs

```diff
@@ -24,17 +24,18 @@ namespace Exiled.Events.Patches.Events.Player
     [HarmonyPatch(typeof(RagdollManager), nameof(RagdollManager.SpawnRagdoll))]
     internal static class SpawningRagdoll
     {
-        private static bool Prefix(RagdollManager __instance, ref Vector3 pos, ref Quaternion rot, ref int classId, ref PlayerStats.HitInfo ragdollInfo, ref bool allowRecall, ref string ownerID, ref string ownerNick, ref int playerId)
+        private static bool Prefix(RagdollManager __instance, ref Vector3 pos, ref Quaternion rot, ref Vector3 vel, ref int classId, ref PlayerStats.HitInfo ragdollInfo, ref bool allowRecall, ref string ownerID, ref string ownerNick, ref int playerId)
```

Comment:
The variable name in the patch must match the variable name in the original method.

Reply:
Thanks, I did actually forgot that since the position and rotation are shortened.